config/src/main/java/com/alibaba/nacos/config/server/service/ConfigDetailService.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.config.server.service;
+
+import com.alibaba.nacos.config.server.constant.Constants;
+import com.alibaba.nacos.config.server.model.ConfigInfo;
+import com.alibaba.nacos.config.server.model.Page;
+import com.alibaba.nacos.config.server.service.repository.ConfigInfoPersistService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+
+/**
+ * config detail service.
+ *
+ * @author 985492783@qq.com
+ * @date 2023/2/9 5:25
+ */
+@Service
+public class ConfigDetailService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfigDetailService.class);
+    
+    private ConfigInfoPersistService configInfoPersistService;
+    
+    private BlockingQueue<SearchEvent> eventLinkedBlockingQueue = new LinkedBlockingQueue<>();
```

Comment:
LinkedBlockingQueue default size is Integer.MAX. 
Should we limit it as configurable or set a little value without configurable?